theano/tensor/opt.py

```diff
@@ -2071,6 +2073,7 @@ class Assert(T.Op):
 
     Examples
     --------
+    import theano
     T = theano.tensor
```

Comment:
This example don't show well on the generated doc. It show  all the code on a single line.